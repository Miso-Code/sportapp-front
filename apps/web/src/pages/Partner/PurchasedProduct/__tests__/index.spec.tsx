import PurchasedProductPartnerPage from '..'
import { render, RenderResult } from '@testing-library/react'

describe('PurchasedProductPartnerPage', () => {
	let wrapper: RenderResult

	beforeEach(() => {
		wrapper = render(<PurchasedProductPartnerPage />)
	})

	afterEach(() => {
		wrapper.unmount()
	})

	it('should render the component', () => {
		expect(wrapper.container).toMatchSnapshot()
	})
})
