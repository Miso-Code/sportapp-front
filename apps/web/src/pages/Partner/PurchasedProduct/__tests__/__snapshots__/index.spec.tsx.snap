// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PurchasedProductPartnerPage should render the component 1`] = `
<div>
  <div>
    <header
      class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorInherit MuiAppBar-positionStatic bg-white css-1cjrmu8-MuiPaper-root-MuiAppBar-root"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthXl css-19r6kue-MuiContainer-root"
      >
        <div
          class="MuiToolbar-root MuiToolbar-regular css-r6ewbb-MuiToolbar-root"
        >
          <div
            class="MuiBox-root css-1lvtzne"
          >
            <button
              aria-controls="menu-appbar"
              aria-haspopup="true"
              aria-label="navbar.partner.menu"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-sizeLarge css-1pz7awu-MuiButtonBase-root-MuiIconButton-root"
              tabindex="0"
              type="button"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="MenuIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M3 18h18v-2H3zm0-5h18v-2H3zm0-7v2h18V6z"
                />
              </svg>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
          <div
            class="MuiBox-root css-1t6c9ts"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorPrimary MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorPrimary css-1eyyvyt-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              title="navbar.partner.getProducts"
              type="button"
            >
              navbar.partner.getProducts
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorPrimary MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorPrimary css-1eyyvyt-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              title="navbar.partner.createProduct"
              type="button"
            >
              navbar.partner.createProduct
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorPrimary MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorPrimary css-1eyyvyt-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              title="navbar.partner.purchasedProducts"
              type="button"
            >
              navbar.partner.purchasedProducts
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
          <div
            class="MuiBox-root css-2uchni"
          >
            <button
              aria-label="navbar.open.profile"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-av538e-MuiButtonBase-root-MuiIconButton-root"
              data-mui-internal-clone-element="true"
              tabindex="0"
              type="button"
            >
              <div
                class="MuiAvatar-root MuiAvatar-circular css-1wlk0hk-MuiAvatar-root"
              >
                <img
                  alt="Remy Sharp"
                  class="MuiAvatar-img css-1pqm26d-MuiAvatar-img"
                  src="/static/images/avatar/2.jpg"
                />
              </div>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
      </div>
    </header>
     Purchased Product
  </div>
</div>
`;
